modules/nf-core/bowtie/build/main.nf

```diff
@@ -8,20 +8,21 @@ process BOWTIE_BUILD {
         'biocontainers/bowtie:1.3.0--py38hed8969a_1' }"
 
     input:
-    path fasta
+    tuple val(meta), path(fasta)
 
     output:
-    path 'bowtie'       , emit: index
-    path "versions.yml" , emit: versions
+    tuple val(meta), path('bowtie') , emit: index
+    path "versions.yml"             , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${fasta.baseName}"
```

Comment:
why not use meta.id?

Reply:
All I did was add the meta at the moment, didn't look at the rest of the tool.